dm.tex

```diff
@@ -42,79 +42,84 @@ \subsubsection{Approach to verification and validation}\label{sec:dm-approach}
 	\item  "Should be done to enter Operations; but waiver likely to be granted if not met," i.e., we could enter Operations without this fulfilled, for first 3 years.
 	\item  "Overall capability/efficiency/ease of use/etc., may be reduced but science will not critically suffer if not done." Could enter operations without this requirement fulfilled, and have the soperations team decide whether they want to pursue it.
 \end{enumerate}
- 
+
+The verification status of requirements in the DMSR will be reported at each of the Construction Closeout Reviews.
+Priority 1a requirements are expected to be fully verified by CCR1, coinciding with the start of on-sky commissioning.
+Priority 1b requirements are expected to be fully verified by CCR3 to demonstrate readiness for the handover to Operations.
+Most of the priority 2 requirements will be verified by CCR4 in the course of demonstrating the capability to produce, document, and serve scientifically validated survey-scale data products to the science community via the Early Science Program \citeds{rtn-011}.
```

Comment:
No - some will be but I'm not expecting to make substantial progress between CCR3 and CCR4. CCR4 is a formality.